posix/posix.c

```diff
@@ -826,12 +904,14 @@ int posix_pipe(int fildes[2])
 	fildes[0] = 0;
 
 	proc_lockSet(&p->lock);
-	while (p->fds[fildes[0]].file != NULL && fildes[0]++ < p->maxfd);
+	while (p->fds[fildes[0]].file != NULL && fildes[0]++ < p->maxfd) {
+	}
 
 	fildes[1] = fildes[0] + 1;
-	while (p->fds[fildes[1]].file != NULL && fildes[1]++ < p->maxfd);
+	while (p->fds[fildes[1]].file != NULL && fildes[1]++ < p->maxfd) {
```

Comment:
i think that using increment in while condition is not compliant to misra rule 13.3, however I am not sure if I interpret that rule right

while loop may instead be changed to a for loop this way it will be compliant for sure and clearer imo